<launch>
				<!--  ************** Odometry ***************  -->
	<arg name="gui" default="True" />
	<param name="use_gui" value="$(arg gui)"/>
	<param name="robot_description" command="cat $(find nox_description)/urdf/nox.urdf" />
	
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
	
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
	
	<node name="serial_node" pkg="rosserial_python" type="serial_node.py">
		<param name="port" value="/dev/ttyACM0"/>
	</node>

	<node name="imu_node" pkg="mpu_6050_driver" type="imu_node.py" >
  	</node>

	<node name="imu_filter_node_for_orientation" pkg="imu_complementary_filter" type="complementary_filter_node" >
  	</node>
		
	<node name="nox_controller" pkg="nox" type="nox_controller">
		<param name="publish_tf" value="true" />
    		<param name="publish_rate" value="10.0" />
    		<param name="linear_scale_positive" value="1.025" />
    		<param name="linear_scale_negative" value="1.025" />
    		<param name="angular_scale_positive" value="1.078" />
    		<param name="angular_scale_negative" value="1.078" />
		<param name="angular_scale_accel" value="0.0" />
                <param name="alpha" value="0.0" />
                <param name="use_imu" value="true" />
	</node>	
	
				<!--  ************** Sensors ***************  -->

	<include file="$(find rplidar_ros)/launch/rplidar.launch" />

	
</launch>
